modules/mobile/app/models/mobile/v0/claim_event_timeline.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'common/models/resource'
+
+module Mobile
+  module V0
+    class ClaimEventTimeline < Common::Resource
+      attribute :type, Types::String
+      attribute :tracked_item_id, Types::Integer.optional
```

Comment:
I think we can add default values here? If I'm not mistaken, the response is intended to always contain each of these attributes, they're just marked optional because they may be nil. If we set the default to nil, it would make instantiating the `ClaimEventTimeline` less tedious. For instance, 

```
            ClaimEventTimeline.new(
               type: :other_documents_list,
               date: nil,
               tracked_item_id: document['trackedItemId'],
               description: nil,
               display_name: nil,
               overdue: nil,
               status: nil,
               uploaded: nil,
               uploads_allowed: nil,
               opened_date: nil,
               requested_date: nil,
               received_date: nil,
               closed_date: nil,
               suspense_date: nil,
               documents: nil,
               upload_date: document['uploadDate'],
               file_type: document['documentTypeLabel'],
               document_type: nil,
               filename: document['originalFileName']
             )
```
Could become 
```
            ClaimEventTimeline.new(
               type: :other_documents_list,
               tracked_item_id: document['trackedItemId'],
               upload_date: document['uploadDate'],
               file_type: document['documentTypeLabel'],
               filename: document['originalFileName']
             )
```

Reply:
Good point. added default values.